serverless/cmd/feeder/main.go

```diff
@@ -90,42 +91,41 @@ func feedLog(l config.Log, w wit_http.Witness, timeout time.Duration, interval t
 		return fmt.Errorf("invalid LogURL %q: %v", l.URL, err)
 	}
 	f := newFetcher(lURL)
+	// TODO(al): make this configurable
```

Comment:
I think this TODO can be dropped. There doesn't seem to be any pretense of flexibility of hasher in the serverless implementation at the moment.